src/Propel/Generator/Builder/Om/TableMapBuilder.php

```diff
@@ -173,8 +173,6 @@ protected function addClassBody(&$script)
             $this->addGetTableMap($script);
         }
 
-        $this->addBuildTableMap($script);
```

Comment:
Can we clarify the BC breaking impact of removing this part?
Also looking into projects that extend the table map builder etc.

Reply:
That whole thing is pretty much internal Propel business. There is no designated way to override table map classes, and the way they are tightly coupled with the base model- and query-classes should make it incredibly difficult to do so.
Also, changing this method would have broken the initialization, unless the whole mechanism was rewritten to an extend, where that Propel instance cannot be updated anymore.
I am pretty sure we are safe here.